src/App.vue

```diff
@@ -1,10 +1,31 @@
-<script setup lang="ts">
-import { RouterView } from 'vue-router'
-</script>
-
 <template>
   <RouterView />
 </template>
 
+<script setup lang="ts">
+import { RouterView } from 'vue-router'
+</script>
+
 <style>
+*,
+*::before,
+*::after {
+  @apply box-border p-0 m-0;
+}
```

Comment:
Is it needed? I think tailwind does that by default

Reply:
https://tailwindcss.com/docs/preflight - apparently so.

However, it doesn't seem to put `margin: 0;` and `padding: 0` on all elements as well as I can't see the `box-sizing` mentioned in here so I just commented my reset out to check if anything changes and it doesn't look like it does. I'm still not 100% sure if nothing breaks later on but if it does, I'll fix it. So, as for now, **I removed this code**.